pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -35,9 +35,9 @@ func (d *DualWriterMode1) Mode() DualWriterMode {
 
 // Create overrides the behavior of the generic DualWriter and writes only to LegacyStorage.
 func (d *DualWriterMode1) Create(ctx context.Context, original runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	log := d.Log.WithValues("kind", options.Kind)
-	ctx = klog.NewContext(ctx, log)
 	var method = "create"
+	log := d.Log.WithValues("kind", options.Kind, "method", method)
```

Comment:
good point! But actually, I might have a better option, let me just revisit this

Reply:
ah, my bad, I thought you meant on the `compare` fn: name would actually be very long (and unique), so with less chances of grouping it, so I had decided by kind, as it could be more interesting to see how latency varies for different entities.

Reply:
You should still be able to add name as a value here, these values don't get turned into loki labels, just k=v entries in the log output

Reply:
I guess I could... I was on the fence about this because not in every method we have access to the name and I was wanting to make every log a bit consistent. But it could be helpful for troubleshooting.

Reply:
yeah, much better to have it wherever it's available than to not have it anywhere